vm-image-service/README.md

```diff
@@ -49,14 +27,37 @@ The UI allows users to execute all necessary workflows:
         - Show details for the Content Library
     - Grid action menu
         - View the YAML for a Content Library
+- Publish VM to a Content Library
 
 ## List all Virtual Machine Images
+![VM Image Card](source/images/vm-image-card.png "VM Image Card")
+VM Image service can be accessed via a card on the namespace summary view. If VM Image service is available for the current namespace then the above shown card will appear in the namespace summary view. Check out the support section in this document.
+
+![VM Image List View](source/images/vm-image-image-list.png "VM Image List View")
+In the above picture, we are looking at the list of VirtualMachine Images which are available for the current namespace. Namespaces are not directly associated with VM Images rather transitively associated through Content Libraries. The above list view is an aggregation of all the VM Images across all Content Libraries that are associated with the current namespace.
 
 ### Actions
+![VM Image Delete](source/images/vm-image-delete.png "VM Image Delete")
+The only user action available for consumption users to perform on a VM Image is "Delete" a VM Image from a namespace-scoped Content Library that has been marked as writable. Delete action will be left unavailable for VM Images which belong to cluster-scoped and read-only libraries.
```

Comment:
I was going to say that "the same way we explain that VM images are consumed via CL, we should explain that there are 2 types of CL: the Supervisor-scoped one and the namespace-scoped one" but I see you explain it later. I think you should explain it before since here you're mentioning "namespace-scoped"

Reply:
I think adding a new section to explain the two scopes should take care of this comment. Please let me know if you think otherwise.